README.md

```diff
@@ -2,7 +2,18 @@
 
 __`lava-dl`__ is a library of deep learning tools within Lava that support  offline training, online training and inference methods for various Deep Event-Based Networks.
 
-The library presently consists of 
+There are two main strategies for training Deep Event-Based Networks: _direct training_ and _ANN to SNN converison_. 
+
+Directly training the network utilizes the information of precise events in time. It is very accurate and results in efficient networks. However, the training these networks take a lot of time and resources.
+
+On the other hand, ANN to SNN conversion is especially suitable for rate coded SNNs where we can leverage the fast training of ANN. These converted SNNs, however, require increased latency compared to directly trained SNNs.
+
+Lava-DL provides improved [SLAYER](https://github.com/bamsumit/slayerPytorch) for direct training of deep event based networks and a new ANN-SNN accelerated training approach called [Bootstrap:TODO](link_here) to mitigate low latency issue of conventional ANN-SNN methods for training Deep Event-Based Networks.
```

Comment:
... provides an improved version of SLAYER....

Reply:
low latency issue? low latency would be good and does not have to be mitigated.

... to mitigate THE issue of long training times.
I would not call it training latency.

Reply:
Should have been high latency issue!